pkg/agent/datamodel/types.go

```diff
@@ -537,6 +537,7 @@ type KubernetesConfig struct {
 	NetworkMode                       string            `json:"networkMode,omitempty"`
 	ContainerRuntime                  string            `json:"containerRuntime,omitempty"`
 	MaxPods                           int               `json:"maxPods,omitempty"`
+	IPFamilies                        []string          `json:"ipFamilies,omitempty"`
```

Comment:
when / where do you set KubernetesConfig.IPFamilies to v4 and v6?

Reply:
all of these values get set in aks-rp. there is a agent baker bootstrap config that gets built by rp and then passed to agent baker during vmss create/update